crypto/asn1/ameth_lib.c

```diff
@@ -9,6 +9,7 @@
 
 #include <stdio.h>
 #include "internal/cryptlib.h"
+#include "e_os.h"
```

Comment:
Not removed.  Needed on Windows for strncasecmp.
I've added a comment to indicate this.